npm/private/npm_link_package_store.bzl

```diff
@@ -150,9 +150,10 @@ npm_link_package_store = rule(
 
 # A private util method to minimize the generated code for local package store links.
 # May be changed/deleted at any time to minimize code from the generated npm_link_all_packages().
-def npm_local_link_package_store_internal(name, src, link_visibility = ["//visibility:public"]):
+def npm_local_link_package_store_internal(name, src, package = None, link_visibility = ["//visibility:public"]):
     npm_link_package_store(
         name = name,
+        package = package,
```

Comment:
NOTE: `npm_link_package_store` creates a `node_modules/{package}` directory, but will default to `src[NpmInfoThing].package` if this attribute is unset. In the case of references to packages by an alias this is required.

NOTE 2: it's very confusing that `name = "node_modules/{package}"`. Maybe that name (when generated by `npm_translate_lock`) should be `link_{i}` or `link_{package}_as_{alias}` instead?

Reply:
Partially moved to https://github.com/aspect-build/rules_js/pull/2474